macros/on-run-end/print_dbt_project_evaluator_issues.sql

```diff
@@ -33,11 +33,13 @@
         {{ print("\n-- " ~ result.node.alias ~ " --") }}
 
         {% set model_checked = result.node.depends_on.nodes[0].split('.')[-1] %}
-        {% set db_schema = database_package ~ "." ~  schema_package if database_package else schema_package %}
-        {% set db_schema_model = db_schema ~ "." ~  model_checked if db_schema else model_checked %}
+        {% set model_details = graph.nodes.values() | selectattr("name","==",model_checked) | first %}
+        {% set model_schema = model_details.schema %}
+        {% set model_database = model_details.database %}
+        {% set db_schema = model_database ~ "." ~ model_schema if model_database else model_schema %}
 
         {% set sql_statement %}
-        select * from {{ db_schema_model }}
+        select * from {{db_schema}}.{{model_checked}}
```

Comment:
worth including the database name here too?

Reply:
`db_schema` already contains `<optional_database>.schema` if a database is defined. But before we merge this I want to try a new, less hacky, way of getting the table to query, I will move this PR as a draft for now.